build-farm/platform-specific-configurations/windows.sh

```diff
@@ -27,21 +27,27 @@ TOOLCHAIN_VERSION=""
 
 # Any version above 8 (11 for now due to openjdk-build#1409
 if [ "$JAVA_FEATURE_VERSION" -gt 11 ]; then
-    BOOT_JDK_VERSION="$((JAVA_FEATURE_VERSION-1))"
+    if [ "$ARCHITECTURE" == "aarch64" ]; then
+      # Windows aarch64 cross compiles requires same version boot jdk
+      BOOT_JDK_VERSION="$((JAVA_FEATURE_VERSION))"
```

Comment:
Are we always cross-compiling though?  I guess we are for now, but would be good to separate that case.